modules/homepage/src/main/frontend/src/themePage/Navbars/SearchBar.jsx

```diff
@@ -251,9 +252,10 @@ function SearchBar(props) {
                       key={i}
                       disabled={result["disabled"]}
                       onClick={(e) => {
+                        const anchor = result[LFS_QUERY_MATCH_KEY][LFS_QUERY_QUESTION_KEY].replace(/\s/g, '').substring(0, 10);
                         // Redirect using React-router
                         if (result["@path"]) {
-                          props.history.push("/content.html" + result["@path"]);
+                          props.history.push("/content.html" + result["@path"] + "#" + anchor);
```

Comment:
Should use the hash argument:
```suggestion
                          props.history.push({
                          pathname: "/content.html" + result["@path"],
                          hash: anchor
                          });
```